docker/startup

```diff
@@ -0,0 +1,14 @@
+#!/bin/sh
+
+RCLONE_CONFIG=/config/rclone/rclone.conf
+
+mkdir -p "$(dirname "$RCLONE_CONFIG")"
+
+cat <<EOF >"$RCLONE_CONFIG"
+[$REMOTE_NAME]
+type = $REMOTE_TYPE
+url = $REMOTE_URL
+vendor = $REMOTE_VENDOR
+EOF
+
+rclone serve s3 "$REMOTE_NAME": --addr :"$PROXY_PORT"
```

Comment:
should we allow to add other options like vfs settings?

Reply:
We can make the lists we want to be configurable and append them in the command. What options we want to be configurable?

Reply:
>for nextcloud I have to set nextcloud_chunk_size = 0 otherwise rclone refuses to work

^